README.md

```diff
@@ -115,6 +115,12 @@ Some of these depend on each other
           backup_password: "sample backup password"
           version: "3.2" # can be 3.0 or 3.2
         ```
+* LEMP
+* postfix
+    ```
+    postfix:
+        mailname: "{{ ansible_hostname }}"
```

Comment:
Please do, vars in defaults are the ones the end user can modify, so it's useful.